doc/github.md

```diff
@@ -63,3 +63,15 @@ The [NixOS foundation board](https://nixos.org/community/teams/foundation-board/
 The home of the branding guide and media kit.
 
 The [marketing team](https://nixos.org/community/teams/marketing/) controls this repository.
+
+## Groups
+
+### [nixpkgs-committers](https://github.com/orgs/NixOS/teams/nixpkgs-committers)
+
+Documented in [./nixpkgs-committers.md](./nixpkgs-committers.md).
+
+### [nixpkgs-maintainers](https://github.com/orgs/NixOS/teams/nixpkgs-maintainers)
+
+A very large group of contributors established by [RFC 39](https://github.com/NixOS/rfcs/blob/master/rfcs/0039-unprivileged-maintainer-teams.md).
+
+Since this is a very large group, _team notifications_ are turned off.
```

Comment:
Everything except the team notifications was already documented in this very file :sweat_smile:  https://github.com/NixOS/org/blob/main/doc/github.md#nixpkgs